react-front-end/__tests__/tsrc/search/components/GallerySearchResult.test.tsx

```diff
@@ -120,5 +134,26 @@ describe("<GallerySearchResult />", () => {
         });
       }
     );
+
+    it.each([
+      ["item", languageStrings.searchpage.gallerySearchResult.viewItem],
+      ["attachment", languageStrings.searchpage.gallerySearchResult.ariaLabel],
```

Comment:
The `languageStrings` have been de-structured up the top, so:

```suggestion
      ["item", viewItem],
      ["attachment", ariaLabel],
```

That `ariaLabel` though keeps grabbing my attention. At somepoint we may want to refactor that with a more useful name - or maybe just in the de-structured context it loses meaning.

Reply:
Perhaps rename to `viewAttachment` ?